### PR TITLE
add upgrade-parathread method at parachain ts-test

### LINKED ISSUE #1097: Parachain onboarding fails with polkadot-v0.9.33
### Context
Steps to reproduce:
- change the following line to download 0.9.33 polkadot binary: https://github.com/litentry/litentry-parachain/blob/8b7f31b764f988b77bda6b27d4e4a796c95923bc/scripts/launch-local-binary.sh#L45
- make launch-binary-rococo
- observe that the `register-parachain` is executed successfully but the parachain can't be onboarded as parachain, but only as parathread:
image

polkadot 0.9.32 binary works smoothly.

---
:heavy_check_mark: Please set appropriate **labels** and **assignees** if applicable.

### PR DESCRIPTION
**Pull request by Marky-Shi**
Resolves #1097

As the title when launch local pararchain by useing binary  at polkadot-v0.9.33 or  higher, parachains cannot produce blocks normally, and you need to manually renew the lease period. 

The image below is the result of my local testing.

image

image

image

image

### CODE REVIEW

**Comment by Marky-Shi** [comment]
There are still small things that have not been dealt with.

File: ts-tests/tests/upgrade-parathread.test.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import '@polkadot/api-augment';
```
Comment (Kailai-Wang) [inline ts-tests/tests/upgrade-parathread.test.ts]: Maybe simply rename it to `upgrade-parathread.ts`, as it's not test

File: scripts/launch-local-binary.sh [hunk 2]
```diff
@@ -105,11 +104,22 @@ $PARACHAIN_BIN --alice --collator --force-authoring --tmp --chain $CHAIN-dev \
...
 echo "NODE_ENV=ci" > .env
 yarn
-yarn register-parachain 2>&1 | tee "$TMPDIR/register-parachain.log"
+yarn register-parathread 2>&1 | tee "$TMPDIR/register-parathread.log"
+print_divider
+
+echo "start upgrading parathread to parachain now, Please wait a moment"
+# Wait for the parathread to complete the registration
+# Wait for 90s because need to wait 1 minute to confirm after registering parathread,
+# so set the waiting time to 90s to ensure normal upgrades.
```
Comment (Kailai-Wang) [inline scripts/launch-local-binary.sh:117]: Yes it's important to have some comments, good one 👍

File: scripts/launch-local-binary.sh [hunk 3]
```diff
@@ -105,11 +104,22 @@ $PARACHAIN_BIN --alice --collator --force-authoring --tmp --chain $CHAIN-dev \
...
+sleep 90
+cd "$ROOTDIR/ts-tests"
+echo "NODE_ENV=ci" > .env
+yarn
+yarn upgrade-parathread 2>&1 | tee "$TMPDIR/upgrad-parathread.log"
```
Comment (Kailai-Wang) [inline scripts/launch-local-binary.sh]: typo? `upgrade-parathread.log`

**Review by Kailai-Wang — APPROVED** [review]
Looks good, thanks. Only a few small things.

I'm still wondering why `sudoScheduleParaInitialize` alone doesn't do the job (anymore)

File: ts-tests/tests/upgrade-parathread.test.ts (hunk 1 above)
Comment (Marky-Shi) [reply ts-tests/tests/upgrade-parathread.test.ts]: okay  got it.

File: scripts/launch-local-binary.sh (hunk 3 above)
Comment (Marky-Shi) [reply scripts/launch-local-binary.sh]: yeah  typo ....

**Comment by Marky-Shi** [comment]
> Looks good, thanks. Only a few small things.
> 
> I'm still wondering why `sudoScheduleParaInitialize` alone doesn't do the job (anymore)

I will investigate the reasons behind this, and if there are results, I will share it in the corner.

**Pull request merged**